posts/ethereum-development-tips.md

```diff
@@ -0,0 +1,254 @@
+---
+authors:
+- Louis Fournier
+- Hugo de Almeida Rodrigues
+tags:
+- Blockchain
+- Ethereum
+- Dai
+- Stablecoin
+- Solidity
+- Truffle
+- Remix
+- ZeppelinOS
+- web3
+- Metamask
+- geth
+date: 
+title: "Ethereum Development Tips"
+image: 
+---
+
+{{< responsive-figure src="/images/2018/08/blockchain-interns-01.png" >}}
+(source: https://coin24.fr/2018/03/07/faut-vraiment-acheter-ethereum-2018/)
+
+# Summary
+
+When we arrived at Ippon USA for our summer internship, we had little knowledge about blockchains. The goal of this article is to give you the basic tools one can use to build a decentralized application, and some tips about the language we used: Solidity. Of course our list is not exhaustive, and is just here to offer you a peek at Ethereum development.
+
+# Introduction to Blockchains and Ethereum
+
+Let’s start with the Wikipedia definition:
+"A blockchain, originally <b>block chain</b>, is a growing list of records, called blocks, which are linked using cryptography. Blockchains which are readable by the public are widely used by cryptocurrencies. Private blockchains have been proposed for business use."
+
+To put it in layman’s terms, a blockchain is simply a transaction record that is decentralized. It means anybody can store and run a copy of the blockchain (becoming a “node”) and therefore read it.
+
+There is not a single blockchain, but different ones for different purposes. The Bitcoin blockchain is the most famous one of all, but there is also the widely popular Ethereum blockchain which we use, and many others. These are public, but you can as easily create a private blockchain, which won’t be available to everyone.
+
+As on a classic ledger, you can only add data to a blockchain. To add new pages to the ledger or new blocks to a blockchain “miners” have to solve complicated and costly algorithm problems. Your transaction will be validated after a few blocks have been mined. This is why transactions cost a fee to reward miners.
+
+Finally, security is a major asset of blockchains, with the help of high-end cryptography and the distributed network effect.
+
+{{< responsive-figure src="/images/2018/08/blockchain-interns-02.png" >}}
```

Comment:
Use `![text](link to image)`

Reply:
In that case, the README needs to be updated... I will submit PR for README when I have time.